.gitignore

```diff
@@ -1,3 +1,8 @@
 node_modules/
 build/
 PowerShell/.DS_Store
+PowerShell/JumpCloud Module/.DS_Store
+PowerShell/JumpCloud Module/Docs/.DS_Store
+PowerShell/JumpCloud Module/en-Us/.DS_Store
+.vscode/settings.json
+.vscode/settings.json
```

Comment:
Am I seeing double?
Also, I'd probably just put in `.DS_Store` and that'll catch them all.